q2_types_genomics/per_sample_data/tests/test_format.py

```diff
@@ -75,6 +77,27 @@ def test_multifasta_manifest_empty(self):
 class TestFormats(TestPluginBase):
     package = 'q2_types_genomics.per_sample_data.tests'
 
+    def test_contig_seqs_dir_fmt_fails_bad_char(self):
+        dirpath = self.get_data_path('bad_char_contigs/')
+        with self.assertRaisesRegex(ValidationError, "Invalid character 'Z'"):
+            fmt_obj = ContigSequencesDirFmt(dirpath, mode='r')
+            fmt_obj.validate()
+
+    def test_contig_seqs_dir_fmt_fails_bad_filename(self):
+        dirpath = self.get_data_path('bad_name_contigs')
+
+        fmt_obj = ContigSequencesDirFmt(dirpath, mode='r')
+        with self.assertRaisesRegexp(
+                ValidationError,
+                "Missing one or more files for ContigSequencesDirFmt"):
+            fmt_obj.validate()
+
+    def test_contig_seqs_dir_fmt_good(self):
+        dirpath = self.get_data_path('good_contigs/')
+        format = ContigSequencesDirFmt(dirpath, mode='r')
+
+        format.validate()
```

Comment:
Is it really necessary to restructure the entire contig data directory just to add one test? There is a test already for asserting the "good contigs" (which you commented out below?). The only other test (the one right above) could have simply be done by pointing to some other existing directory with no format-acceptable files?